src/theme/palette.ts

```diff
@@ -0,0 +1,12 @@
+import { sniffrPrimaryColor, sniffrSecondaryColor } from './custom';
+
+const palette = {
+  primary: {
+    main: sniffrPrimaryColor,
```

Comment:
Do these colors need to be defined & imported? You can access them through the theme later, so I feel like putting them in the theme is defining them well enough. 🤔

Reply:
Also: this is a nitpick, feel free to ignore. :3